executable/Uninstall.re

```diff
@@ -0,0 +1,53 @@
+open Fnm;
+open Lwt;
+
+let run = (~version) => {
+  let versionName =
+    switch (version) {
+    | None =>
+      Logger.log(<Pastel> "You should provide a node version" </Pastel>);
+      exit(1);
+    | Some(versionName) => versionName
+    };
+
+  let%lwt _ = {
+    let%lwt installedVersion =
+      Versions.getInstalledVersions()
+      >|= List.find_opt(x => Versions.Local.(x.name == versionName));
+    switch (installedVersion) {
+    | None =>
+      Logger.log(
+        <Pastel>
+          "The version "
+          <Pastel color=Pastel.Cyan> versionName </Pastel>
+          " of node is not installed."
+        </Pastel>,
+      );
+      exit(1);
+    | Some(installedVersion) =>
+      {
+        Logger.log(
+          <Pastel>
+            "Uninstalling node "
+            <Pastel color=Pastel.Cyan>
+              Versions.Local.(installedVersion.name)
+            </Pastel>
+          </Pastel>,
+        );
+      };
+      let%lwt _ = Fs.rmdir(Versions.Local.(installedVersion.fullPath));
```

Comment:
I wonder if we should also clean up all the aliases that target this version. Also, what do you think about adding `Versions.Local.remove(installedVersion)`?

Reply:
Done.

Reply:
Added `Versions.Local.remove(installedVersion)`.

For the aliases, I would prefer to do it in another PR as I'm not yet sure how it works